app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +240,18 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      {if @state.workflows.length > 0
+        meta = @state.workflows[0].getMeta()
+
+        <Paginator
+          className="lab-visibility-paginator"
```

Comment:
Also, if it makes sense as a basic style, feel free to update the paginator.styl file.